security/pkg/pki/ca/ca.go

```diff
@@ -60,8 +60,10 @@ const (
 	TLSSecretRootCertFile = "ca.crt"
 	// The standard key size to use when generating an RSA private key
 	rsaKeySize = 2048
-	// ExternalCASecret stores the plugin CA certificates, in external istiod scenario, the secret can be in the config cluster.
-	ExternalCASecret = "cacerts"
+	// CACertsSecret stores the plugin CA certificates, in external istiod scenario, the secret can be in the config cluster.
+	CACertsSecret = "cacerts"
```

Comment:
very confusing with `CASecret = "istio-ca-secret"`

Reply:
Good point. Do you have any naming suggestions for CASecert?